### PR TITLE
update node & ruby snippets

### PR DESCRIPTION
**Pull request by bencgreenberg**
Use objects for request body in both the Ruby and Node snippets

### CODE REVIEW

File: src/includes/home/api-example.njk [hunk 1]
```diff
@@ -68,7 +81,20 @@ request = Net::HTTP::Post.new(url)
...
+    activity: {
+        description: 'Getting Started with the Orbit Model',
+        link: 'https://orbit-event.eventbrite.com/',
+        link_text: 'See the event',
+        title: 'Orbit Meetup Registration',
+        activity_type: 'eventbrite:registration',
+        occurred_at: '2020-05-10',
+        member: {
+            github: 'user-github',
+            email: 'user@email'
```
Comment (dzello) [inline src/includes/home/api-example.njk:94]: @bencgreenberg Small tweak - `user@email.com` ?

Comment (bencgreenberg) [reply src/includes/home/api-example.njk:94]: Yes, we can do that. The original snippets didn't have the `.com`. I wasn't sure if that was intentional. 🤷🏻

File: src/includes/home/api-example.njk [hunk 2]
```diff
@@ -43,9 +43,22 @@ let options = {
...
```
Comment (bencgreenberg) [inline src/includes/home/api-example.njk:58]:
```suggestion
            email: 'user@email.com'
```

File: src/includes/home/api-example.njk (hunk 1 above)
Comment (bencgreenberg) [inline src/includes/home/api-example.njk:94]:
```suggestion
            email: 'user@email.com'
```

**Review by phacks — APPROVED** [review]

**Pull request merged**